README.md

```diff
@@ -33,6 +50,8 @@ microservices](./docs/img/architecture-diagram.png)](./docs/img/architecture-dia
 
 Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
+<p style="color:red">[[Does anything here need to change? We should add the type of tracing used in each service]]</p>
```

Comment:
Let's do that as a last step. Let's get some Learning Path material in place first and, based on what we're highlighting, make sure the README appropriately highlights that.